spec/mailers/previews/notification_mailer_preview.rb

```diff
@@ -0,0 +1,9 @@
+class NotificationMailerPreview < ActionMailer::Preview
+  def digest
+    @user = User.first
+    @notifications = @user.notifications.for_digest(1.day.ago)
+    @type = :digest
+    ActionMailer::Base.default_url_options[:host] = 'dradis-framework.dev'
```

Comment:
why is this needed? Isn't it set by smtp.yml + setup_mail initializer?
```suggestion
    ActionMailer::Base.default_url_options[:host] = 'dradisframework.dev'
```

Reply:
local testing in the event the smtp file doesn't exist.